### PR TITLE
Fix allow msg without topup

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/msgs/migrations/0140_fail_msgs_missing_topups.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from django.utils import timezone
+
+
+def fail_msg_without_topup(apps, schema_editor):  # pragma: no cover
+    Msg = apps.get_model("msgs", "Msg")
+
+    now = timezone.now()
+    day_ago = now - timedelta(hours=24)
+
+    msgs = Msg.objects.filter(direction="O", topup=None, created_on__lte=day_ago)
```
Comment (nicpottier) [inline temba/msgs/migrations/0140_fail_msgs_missing_topups.py:15]: Don't we want some status filters in here too? IE, `Q`?

**Review by nicpottier — CHANGES_REQUESTED** [review]

File: temba/msgs/migrations/0140_fail_msgs_missing_topups.py (hunk 1 above)
Comment (nicpottier) [reply temba/msgs/migrations/0140_fail_msgs_missing_topups.py:15]: Looking at TextIt, we also have a bunch of outgoing with a status of `E` without a topup, so should probably include those as well.

Comment (norkans7) [reply temba/msgs/migrations/0140_fail_msgs_missing_topups.py]: Filtered by `status__in=['Q', 'E']` so we do not change the delivered/sent/wired ones,

I limited to a day ago, I think newer messages being resent is fine.

**Review by nicpottier — APPROVED** [review]

File: temba/msgs/migrations/0140_fail_msgs_missing_topups.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.4 on 2020-09-08 17:22
+
+from datetime import timedelta
+
+from django.db import migrations
+from django.utils import timezone
+
+
+def fail_msg_without_topup(apps, schema_editor):  # pragma: no cover
```
Comment (rowanseymour) [inline temba/msgs/migrations/0140_fail_msgs_missing_topups.py:9]: no unit test... bold 😄

**Pull request merged**